eks-hybrid-iam-ra-setup.sh

```diff
@@ -51,6 +67,15 @@ cat << EOF > "${SECRETS_DIR}/node.key"
 ${NODE_KEY_DATA}
 EOF
 
+if [ "${ENABLE_CREDENTIALS_FILE}" = "true" ]; then
+    export EKS_HYBRID_SHARED_CREDENTIALS_FILE="/var/eks-hybrid/.aws/credentials"
+    export SIGNING_HELPER_UPDATE_COMMAND=$(get_aws_signing_helper_update_command ${SECRETS_DIR}/config)
+    envsubst '$EKS_HYBRID_SHARED_CREDENTIALS_FILE:$SIGNING_HELPER_UPDATE_COMMAND' \
```

Comment:
```suggestion
    # shellcheck disable=SC2016 # we want to replace the variables verbatim
    envsubst '$EKS_HYBRID_SHARED_CREDENTIALS_FILE:$SIGNING_HELPER_UPDATE_COMMAND' \
```

Just to keep things clear, it might be nice to add a ShellCheck exception here.

Reply:
Done